tests/Type/PHPUnit/data/assert-function.php

```diff
@@ -20,6 +20,17 @@ public function doFoo($o): void
 		assertType(self::class, $o);
 	}
 
+	/**
+	 * @template T of object
+	 * @param object $o
+	 * @param class-string<T> $class
+	 */
+	public function assertInstanceOfWorksWithTemplate($o, $class): void
+	{
+		assertInstanceOf($class, $o);
+		assertType($class, $o);
```

Comment:
This isn't a valid test case. The first parameter of `assertType` needs to be a literal string like `'Foo'`.